### PR TITLE
Add description about steps to translate product attribute and attribute options in Magento 2

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

This pull request (PR) adds description about steps to translate product attribute and attribute options in Magento 2

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

- no

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

- no

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

- https://docs.magento.com/user-guide/catalog/product-translate.html

whatsnew
Updated the [Translating Products](https://docs.magento.com/user-guide/catalog/product-translate.html) topic to include steps for translating product attributes and attribute options.

### CODE REVIEW

File: src/catalog/product-translate.md [hunk 1]
```diff
@@ -69,3 +69,19 @@ If your store has multiple views in different languages, the same products are a
 1. When complete, click <span class="btn">Save Category</span>.
 
 1. Repeat the process for all categories used in the store.
+
+## Step 4: Translate product attributes and attributes options
+
+1. Navigate to _Stores > Attributes > Product_ and select the attribute to be translated.
```
Comment (leslietilling) [inline src/catalog/product-translate.md:75]:
```suggestion
1. On. the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.

1. Select the attribute to be translated.
```

File: src/catalog/product-translate.md [hunk 2]
```diff
@@ -69,3 +69,19 @@ If your store has multiple views in different languages, the same products are a
...
+
+1. Under the **Manage Label** section, set the translated attribute title.
```
Comment (leslietilling) [inline src/catalog/product-translate.md:77]:
```suggestion
1. Choose **Manage Label** on the left and set the **Managed Titles** options to define the attribute title translations.
```

File: src/catalog/product-translate.md [hunk 3]
```diff
@@ -69,3 +69,19 @@ If your store has multiple views in different languages, the same products are a
...
+
+    ![]({% link images/images/manage-label-tab.png %}){: .zoom}
+    _Manage Label tab_
```
Comment (leslietilling) [inline src/catalog/product-translate.md:80]:
```suggestion
    _Manage Titles_
```

File: src/catalog/product-translate.md [hunk 4]
```diff
@@ -69,3 +69,19 @@ If your store has multiple views in different languages, the same products are a
...
+
+1. Under _Properties > Manage Options (Values of Your Attribute)_, translate the attribute options
```
Comment (leslietilling) [inline src/catalog/product-translate.md:82]:
```suggestion
1. Choose **Properties** on the left and enter the translated attribute options in the **Manage Options** section.
```

File: src/catalog/product-translate.md [hunk 5]
```diff
@@ -69,3 +69,19 @@ If your store has multiple views in different languages, the same products are a
...
+
+    ![]({% link images/images/manage-option-tab.png %}){: .zoom}
+    _Manage Option tab_
```
Comment (leslietilling) [inline src/catalog/product-translate.md:85]:
```suggestion
    _Manage Options_
```

**Review by leslietilling — CHANGES_REQUESTED** [review]
Minor edits for style and clarity

**Comment by EvheniiaKolkhiieva** [comment]
Hi @leslietilling , thank you for the rewiew. I have implemented your suggestions, please approve the PR. Thank you! 😊

File: src/catalog/product-translate.md (hunk 5 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-translate.md]: Done ✅

File: src/catalog/product-translate.md (hunk 4 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-translate.md]: Done ✅

File: src/catalog/product-translate.md (hunk 3 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-translate.md]: Done ✅

File: src/catalog/product-translate.md (hunk 2 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-translate.md]: Done ✅

File: src/catalog/product-translate.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/catalog/product-translate.md]: Done ✅

File: src/catalog/product-translate.md [hunk 6]
```diff
@@ -69,3 +69,21 @@ If your store has multiple views in different languages, the same products are a
 1. When complete, click <span class="btn">Save Category</span>.
 
 1. Repeat the process for all categories used in the store.
+
+## Step 4: Translate product attributes and attributes options
+
+1. On. the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
```
Comment (leslietilling) [inline src/catalog/product-translate.md:75]: This was my typo in comments
```suggestion
1. On the _Admin_ sidebar, go to **Stores** > _Attributes_ > **Product**.
```

**Review by leslietilling — APPROVED** [review]

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by leslietilling** [comment]
@magento import pr to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@leslietilling the pull request successfully imported.

**Pull request merged**